mapiproxy/services/web/rpcproxy/rpcproxy/channels.py

```diff
@@ -447,8 +457,7 @@ def _wait_IN_channel(self):
 
             connected = True
         else:
-            self.logger.info("IN channel failed to connect in the last"
-                             " 10 seconds")
+            self.logger.info("Failed to connect to IN channel in last 10 seconds")
```

Comment:
I think the previous message was more correct, you are connected to the IN channel, but waiting for it to connect to samba